### DISCUSSION TITLE
Welcome to LightGBMLSS Discussions!

### DISCUSSION THREAD

**Discussion by StatMixedML** [StatMixedML/LightGBMLSS#8] [Announcements]
## 👋 Welcome!
Dear Community,

In case you find the LightGBMLSS library interesting for your research / use-case, please use this forum for posting new feature requests you would like to see in the future.

**Comment by tkprochman** [comment]
what is the current status for zero-inflated?

**Comment by StatMixedML** [comment]
@tkprochman Thanks for your interest in the project. Right now there are no plans to add additional distributions, but feel free to raise a PR with a zero-inflated distribution.
